app/csi.go

```diff
@@ -51,7 +46,6 @@ func runCSI(c *cli.Context) error {
 	return manager.Run(c.String("drivername"),
 		c.String("nodeid"),
 		c.String("endpoint"),
-		c.String("csi-version"),
 		identityVersion,
```

Comment:
This version is always `dev`. Do we need to retain it?

Reply:
I think it is the same as the version in Longhorn manager image (e.g., v1.1.2, v1.2.0, v1.2.1-rc2)

Reply:
I guess dapper is doing some magic here to set it https://github.com/longhorn/longhorn-manager/blob/532d53d1f1d1484abfff0e1e8a440d5bb18798bf/Dockerfile.dapper#L30

Reply:
OK. You are right. Then this field looks good.